content/en/universal_service_monitoring/setup.md

```diff
@@ -931,6 +937,63 @@ agents:
 {{% /tab %}}
 
 {{< /tabs >}}
+{{< /collapse-content >}}
+
+{{< collapse-content title="Universal Kafka Monitoring (Private Beta)" level="h4" >}}
+
+<div class="alert alert-info">
+Universal Kafka Monitoring is available in <strong>Private beta</strong>.
+</div>
+
+<strong>Note</strong>:
+<br>
+<ul role="list">
+  <li>Producers and consumers require Linux Kernel version 5.2 or later.</li>
+  <li>Producers and consumers must be interfacing with Kafka <strong>without</strong> TLS.</li>
+  <li>Datadog recommends Helm with Datadog chart version 2.26.2 or later. The chart version is defined in the <a href="https://github.com/DataDog/helm-charts/blob/main/charts/datadog/Chart.yaml">`chart.yaml`</a> file of the dd-agent.</li>
+  <li>Requires Agent version 7.53 or greater.</li>
+</ul>
+
+{{< tabs >}}
+{{% tab "Configuration file" %}}
+
+Add the following configuration to the `system-probe.yaml`:
+
+```yaml
+service_monitoring_config:
+  enabled: true
+  enable_kafka_monitoring: true
+```
+
+{{% /tab %}}
+{{% tab "Environment variable" %}}
+
+```conf
+DD_SERVICE_MONITORING_CONFIG_ENABLE_KAFKA_MONITORING=true
+```
+{{% /tab %}}
+
+{{% tab "Helm" %}}
+
+```conf
+datadog:
+  ...
+  serviceMonitoring:
+    enabled: true
+
+agents:
+  ...
+  containers:
+    systemProbe:
+      env:
+        - name: DD_SERVICE_MONITORING_CONFIG_ENABLE_KAFKA_MONITORING
+          value: " true"
```

Comment:
This seems like a strange value, should it just be `true`? Or have the space removed?

Reply:
@conward2121 Is this value correct?

Reply:
```suggestion
          value: "true"
```